<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(45, 45, 45);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="TopBar">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>45</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(37, 37, 37);
border-bottom-color: rgb(20, 20, 20);
color: rgb(231, 231, 231);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="FrameToggle">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"> QPushButton {
	color: rgb(231, 231, 231);
    background-color:rgb(170, 85, 255);
	border: 0px solid;
 }
 QPushButton:hover {
    background-color: rgb(190, 105, 255);
 }
 QPushButton:pressed{
    background-color:rgb(210, 135, 255) ; 
 }</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="ButtonToggle">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Fold</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../DeepPen/icons/deployArrow.png</normaloff>../DeepPen/icons/deployArrow.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignLeft">
        <widget class="QLabel" name="_">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="SectionTitle">
         <property name="font">
          <font>
           <pointsize>17</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Startup</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignRight">
        <widget class="QFrame" name="frame">
         <property name="styleSheet">
          <string notr="true"> QPushButton {
	color: rgb(231, 231, 231);
    background-color:rgb(25, 25, 25);
	border: 2px solid;
	border-color:rgb(55, 89, 55);
 }
 QPushButton:hover {
    background-color: rgb(45, 45, 45);
 }
 QPushButton:pressed{
    background-color:rgb(90, 90, 90) ;
 }

 border-width:1px;
 border-radius:17.2px;</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="FrameTop">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label">
               <property name="font">
                <font/>
               </property>
               <property name="text">
                <string>Current Status:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="ConectedStatusRB">
               <property name="font">
                <font/>
               </property>
               <property name="styleSheet">
                <string notr="true">QRadioButton:checked {
    color: rgb(0, 255, 127);
}
QRadioButton:unchecked {
    color: rgb(255, 64, 64);
}

QRadioButton::indicator {
    width: 10px;
    height: 10px;
    border-radius: 7px;
}

QRadioButton::indicator:checked {
    background-color: rgb(0, 255, 127);
    border: 2px solid  rgb(35, 35, 35);
}

QRadioButton::indicator:unchecked {
    background-color: rgb(255, 64, 64);
    border: 2px solid  rgb(35, 35, 35);
}</string>
               </property>
               <property name="text">
                <string>Disconected</string>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
               <property name="autoRepeat">
                <bool>false</bool>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ButtonSync">
            <property name="font">
             <font/>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../DeepPen/icons/sync.png</normaloff>../DeepPen/icons/sync.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="Content">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="FrameSidebar">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(40, 40, 40);
border-right-color: rgb(20, 20, 20);
</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignTop">
           <widget class="QFrame" name="FrameTopSidebar">
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true"> QPushButton {
	color: rgb(231, 231, 231);
    background-color:rgb(35, 35, 35);
	border: 0px solid;
	text-align: left;
	padding-left: 10px;
 }
 QPushButton:hover {
    background-color: rgb(45, 45, 45);
 }
 QPushButton:pressed{
    background-color:rgb(90, 90, 90) ;
 }</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>1</number>
             </property>
             <property name="leftMargin">
              <number>1</number>
             </property>
             <property name="topMargin">
              <number>1</number>
             </property>
             <property name="rightMargin">
              <number>1</number>
             </property>
             <property name="bottomMargin">
              <number>1</number>
             </property>
             <item>
              <widget class="QPushButton" name="ButtonReadMode">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>35</height>
                </size>
               </property>
               <property name="font">
                <font/>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Letter by...    </string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../DeepPen/icons/letter_shifted.png</normaloff>../DeepPen/icons/letter_shifted.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>35</width>
                 <height>35</height>
                </size>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignTop">
              <widget class="QPushButton" name="ButtonBlackboard">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>35</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <italic>false</italic>
                 <bold>false</bold>
                 <strikeout>false</strikeout>
                </font>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Blackboard...</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>35</width>
                 <height>35</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ButtonAboutDev">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font/>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true"> QPushButton {
	color: rgb(231, 231, 231);
    background-color:rgb(35, 35, 35);
	border: 0px solid;
	padding-left: 10px;
 }
 QPushButton:hover {
    background-color: rgb(45, 45, 45);
 }
 QPushButton:pressed{
    background-color:rgb(90, 90, 90) ;
 }</string>
            </property>
            <property name="text">
             <string>About...</string>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="FramePages">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QStackedWidget" name="stackedWidget">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <widget class="QWidget" name="StartupPage">
             <property name="font">
              <font>
               <bold>false</bold>
              </font>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_6">
                  <property name="spacing">
                   <number>5</number>
                  </property>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_7">
                    <item>
                     <widget class="QLabel" name="StartupTitle">
                      <property name="enabled">
                       <bool>true</bool>
                      </property>
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="font">
                       <font>
                        <pointsize>20</pointsize>
                        <italic>true</italic>
                        <bold>true</bold>
                       </font>
                      </property>
                      <property name="layoutDirection">
                       <enum>Qt::LeftToRight</enum>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">color: rgb(231, 231, 231);
outline:2px black;</string>
                      </property>
                      <property name="text">
                       <string>DeepPen Software</string>
                      </property>
                      <property name="scaledContents">
                       <bool>false</bool>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignCenter</set>
                      </property>
                      <property name="wordWrap">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_6">
                    <item>
                     <widget class="QLabel" name="StartupImage">
                      <property name="minimumSize">
                       <size>
                        <width>270</width>
                        <height>395</height>
                       </size>
                      </property>
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                    <item alignment="Qt::AlignHCenter|Qt::AlignTop">
                     <widget class="QLabel" name="ProjectDescription">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="font">
                       <font/>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">color: rgb(230, 230, 230);</string>
                      </property>
                      <property name="text">
                       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;According to the function the text is supposed to&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;be started from the bottom left but it starts too low and&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;left. I know I can find a point by trial error so it doesn’t gets&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;cut off- you can -20 to the height and it will be fine enough&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;for this one. But it will only fix this specific text! It wont be&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;the same for any label with a different size or text or font.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignJustify|Qt::AlignVCenter</set>
                      </property>
                      <property name="margin">
                       <number>10</number>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
                   <widget class="QLabel" name="DevName">
                    <property name="font">
                     <font>
                      <pointsize>10</pointsize>
                      <italic>true</italic>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">color: rgb(231, 231, 231);</string>
                    </property>
                    <property name="text">
                     <string>By Antonio Priego Raya</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="LetterByLetterPage">
             <widget class="QLabel" name="label_2">
              <property name="geometry">
               <rect>
                <x>370</x>
                <y>110</y>
                <width>141</width>
                <height>101</height>
               </rect>
              </property>
              <property name="text">
               <string>LetterByLetter</string>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="BlackboardPage">
             <widget class="QLabel" name="label_3">
              <property name="geometry">
               <rect>
                <x>400</x>
                <y>40</y>
                <width>101</width>
                <height>41</height>
               </rect>
              </property>
              <property name="text">
               <string>Blackboard</string>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="AboutDevPage">
             <widget class="QLabel" name="label_4">
              <property name="geometry">
               <rect>
                <x>340</x>
                <y>30</y>
                <width>121</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>AboutDev</string>
              </property>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
